### ISSUE TITLE
Anki math

### ISSUE THREAD

**Issue by ajschumacher** [ajschumacher/ajschumacher.github.io#242]
Combine with #230?

“steady state” of reviews when adding N cards per day goes as log(N) ... I have notes in scratch.txt, I think.

also “spike” behavior (reviews per day go to zero)

maybe it would be nice for the scheduler to “smooth out” spiky review days, when you add a ton on one day (which you shouldn’t do, in general)

**Comment by ajschumacher** [comment]
how sustainable is spaced repetition?

say you add 10 cards per day, do perfectly, and us an doubling review
schedule, so you review after one day, 2 days, 4 days, 8 days, etc.
```
 day reviews
 1   10 (10)
 2   10 10 (20)
 3      10 10 (20)
 4   10    10 10 (30)
 5      10    10 10 (30)
 6         10    10 10 (30)
 7            10    10 10 (30)
 8   10          10    10 10 (40)
 9      10          10    10 10 (40)
10         10          10    10 10 (40)
11            10          10    10 10 (40)
12               10          10    10 10 (40)
13                  10          10    10 10 (40)
14                     10          10    10 10 (40)
15                        10          10    10 10 (40)
16   10                      10          10    10 10 (50)
```
so the number of reviews goes as (number added per day) * log_2(days)

for a year, thats n * 8.5 or so (10 * 8.5 = 85 cards/day)
for 10 years, that's about 2^12, so 10 * 12 = 120 cards/day

so if I'm adding 40 cards a day, that's 40 * 8.5 = 340 cards a day

my actual cards added per day is about 18 or 20
and I've been doing it for about 64 days, so log_2(64) = 6
which would suggest 20 * 6 = 120 cards/day
which isn't _that_ far off from where I am
(150 reviews a day if I'm not adding more stuff)

**Comment by ajschumacher** [comment]
I started 2020-10-07 (and have only missed one day so far; wow!) so 336 days, and I have 3812 cards currently. That's an average 11.3 added per day, if they were added evenly over time.

log_2(336) is 8.4

11.3 * 8.4 is 94.9, which is just slightly higher than my actual number of daily reviews recently... Being a little over makes sense maybe because I've been adding fewer cards recently than I was longer ago, I think.
